src/js/core/util/animation/animation.js

```diff
@@ -1,4 +1,4 @@
-/* global requestAnimationFrame, define, ns */
+/* global define, ns */
```

Comment:
In this file have been fixed indentations. You have to change setting in github to better view of changes.

![obraz](https://user-images.githubusercontent.com/29534410/83737868-e2747200-a653-11ea-91ae-b044ffa8c3f1.png)